src/tests/readytorun/coreroot_determinism/coreroot_determinism.csproj

```diff
@@ -1,5 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
   <PropertyGroup>
+    <!-- Needed for CLRTestTargetUnsupported, GCStressIncompatible, CrossGenTest -->
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
     <OutputType>exe</OutputType>
```

Comment:
There are several lowercase exes in this batch that got left behind like before.

Reply:
Fixed in 12th commit.